web/src/hooks/mutations/useConfig.ts

```diff
@@ -0,0 +1,27 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { editProjectConfig } from '../../api/project';
+import { toast } from 'react-hot-toast';
+import { AxiosError } from 'axios';
+
+export const useConfig = (
```

Comment:
this is a good example of what I was describing above. Just pass params, create the mutation, and return